sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -2428,6 +2429,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -IsEnableAppAuthPopUpEnabled
+
+Enables or disables users in the tenancy to authenticate into SharePoint applications using popups. 
```

Comment:
```suggestion
Enables or disables users in the organization to authenticate SharePoint applications using popups. 
```

Reply:
This parameter affects the way code in SharePoint interacts with Azure AD to get token to access APIs. In scenarios where 3rd party cookies are disabled (e.g. Safari browsers with ITP feature enabled), any code that requires a token to access an API automatically triggers a full page refresh. When IsEnableAppAuthPopUpEnabled is set to $true, SharePoint will instead surface a popup if 3rd party cookies are disabled.